examples/css/README.md

```diff
@@ -0,0 +1,168 @@
+# Mistica css
+
+If you can't use the Mistica React components in your project, you can still use the Mistica css to style your
+app.
+
+Mistica css is a set of stylesheets that you can use to style your app with the design tokens of the system.
+It includes css custom properties (css variables) for colors, border radius, typography, etc.
+
+## How to use
+
+### 1. Install the package
+
+```bash
+yarn add @telefonica/mistica
+```
+
+This is the same package which includes the react components, but if you aren't using React, you can just
+import the css files in your app.
+
+### 2. Import the css
+
+You need to import the `mistica-common.css` file and at least one skin file.
+
+For example:
+
+```css
+@import '@telefonica/mistica/css/mistica-common.css';
+@import '@telefonica/mistica/css/movistar.css';
+```
+
+There is one css file for each skin (`movistar`, `o2`, `vivo`, `tu`, `telefonica`, etc)
+
+:warning: The css import method will depend on your bundler or tooling. The files are located inside the
+`css/` folder in the package (`node_modules/@telefonica/mistica/css/`)
+
+### 3. Setup the mistica skin in your html
+
+```html
+<body data-mistica-skin="movistar"></body>
+```
+
+## Example
+
+You can run the example in this folder to see how it works:
+
+```bash
+yarn dev
+```
+
+Take a look at the `index.html` and `styles.css` files.
+
+## Dark mode support
+
+By default, color tokens will change folloging the user's system preferences. If you want to force the dark or
+light mode, you can add the `data-mistica-color-scheme` attribute:
+
+```html
+<body data-mistica-skin="movistar" data-mistica-color-scheme="light"></body>
+```
+
+`data-mistica-color-scheme` can be `light`, `dark` or `auto` (default).
+
+## CSS custom properties
+
+You can use the css custom properties in your css files. For example:
+
+```css
+.my-component {
+  color: var(--mistica-color-textPrimary);
+  background: var(--mistica-color-backgroundContainer);
+  border: 1px solid var(--mistica-color-border);
+  border-radius: var(--mistica-border-radius-container);
+}
+```
+
+In general, the defined custom properties have the following naming convention:
+`--mistica-{category}-{tokenName}`. Categories are `color`, `border-radius`, `font-size`, `font-weight`,
+`line-height`.
+
+To see all the available tokens, you can read the
+[design tokens definition](https://github.com/Telefonica/mistica-design/blob/production/tokens/movistar.json)
+or take a look inside one of the skin css files.
+
+### Color tokens
+
+Naming convention:
+
+- `--mistica-color-{colorName}`
+
+For example:
+
+- `--mistica-color-textPrimary`
+- `--mistica-color-backgroundContainer`, etc.
+
+### Border radius tokens
+
+Naming convention:
+
+- `--mistica-border-radius-{radiusName}`
```

Comment:
I think we decided not to create these tokens, right?

Reply:
hmmm... no, we decided to remove the border-radius utility classes, but we keep the custom properties